creator-node/src/fileManager.js

```diff
@@ -65,14 +65,22 @@ async function saveFileToIPFSFromFS (req, srcPath, fileType, t) {
 
   req.logger.info(`beginning saveFileToIPFSFromFS for srcPath ${srcPath}`)
 
-  const multihash = (await ipfs.addFromFs(srcPath))[0].hash
+  let codeBlockTimeStart = Date.now()
```

Comment:
should we remove this from addFileFromBuffer?

Reply:
(the pin op)

Reply:
I think we can delete. @SidSethi do you agree?